source/common/json/config_schemas.cc

```diff
@@ -1205,6 +1205,10 @@ const std::string Json::Schema::CLUSTER_SCHEMA(R"EOF(
         "enum" : ["http2"]
       },
       "http_codec_options" : {"type" : "string"},
+      "dns_resolvers": {
+        "type" : "array",
```

Comment:
is it a valid configuration to have an empty array? (we can enforce by schema to have at least 1 element if "dns_resolves" is present)